public/admin/config.yml

```diff
@@ -253,6 +253,13 @@ collections:
           - *header_config
           - *meta_config
 
+      - name: "security/bug-bounty"
+        label: "Bug Bounty"
+        file: "src/content/pages/security/bug-bounty.md"
+        fields:
```

Comment:
missing preview path? Think the `generatePageTemplate` should be updated 🤔 It's missing `preview_path: "${pageName}"`
in `newPageConfig`

Reply:
@CharlyMartin I'll open a PR to fix the template

Reply:
Resolved live with @barbaraperic